snowpack/src/commands/build.ts

```diff
@@ -292,7 +292,6 @@ export async function command(commandOptions: CommandOptions) {
   async function installDependencies() {
     const scannedFiles = Object.values(buildPipelineFiles)
       .map((f) => Object.values(f.filesToResolve))
-      .filter(Boolean)
```

Comment:
`Boolean`  must return true since `Object.values` returns a array.

Reply:
`.filter(Boolean)` filters out all falsy values. E.g.

```
> Object.values({key: 'value', emptyValue: ''}).filter(Boolean)
< Array [ "value" ]
```

So I'm not 100% sure about this particular change. Is there a case when a value for the `f.filesToResolve` object can be falsy?

Reply:
```
Object.values(buildPipelineFiles)
      .map((f) => Object.values(f.filesToResolve))
```
this code returns an array of array, so argument passed to Boolean is an array.

Reply:
haha good catch!